hack/jenkins/upload-to-gcs.sh

```diff
@@ -115,11 +121,26 @@ function upload_artifacts_and_build_result() {
       echo "Uploading build log"
       gsutil -q cp -Z -a "${gcs_acl}" "${WORKSPACE}/build-log.txt" "${gcs_build_path}"
     fi
+
+    # For pull jobs, keep a canonical ordering for tools that want to examine
+    # the output.
+    echo "aoeu debug: ${gcs_indirect_path}"
```

Comment:
by the time I got an LG on that PR it was late Friday, and I didn't want to accidentally break Jenkins over the weekend. I'll merge it now.